TxtControllerLib/Components/Sequencer/ControllerSequencer.cs

```diff
@@ -30,7 +30,7 @@ public class ControllerSequencer : IControllerSequencer
         private List<Position> positions;
 
         /// <summary>
-        /// Creates a new instance of the <see cref="ControllerSequencer" /> and starts the communication with the controller. To stop the communication
+        /// Creates a new instance of the <see cref="ControllerCommunicator" /> and starts the communication with the controller. To stop the communication
```

Comment:
Proposal: 
Creates the <see cref="ControllerCommunicator" />, loads the stored positions for usage in the <see cref="MotorPositionController" />s and starts the the communication to the controller.